torchtune/data/_converters.py

```diff
@@ -55,3 +55,59 @@ def sharegpt_to_llama2_messages(
         masked = (role != "assistant") and (not train_on_input)
         messages.append(Message(role=role, content=content, masked=masked))
     return messages
+
+
+def standard_chat_to_llama2_messages(
```

Comment:
Why call this standard chat, and not openai_to_llama2_messages?

Reply:
Will rename! I just came up with this random name for some reason..